src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -58,4 +68,86 @@ static ProcessMetrics()
     public ProcessMetrics(ProcessInstrumentationOptions options)
     {
     }
+
+    private class InstrumentsValues
+    {
+        private const int TicksPerSecond = 10 * 7;
```

Comment:
This doesn't look correct to me. 

https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/TimeSpan.cs,61